.ci/releasenotes-gen-xdoc-push.sh

```diff
@@ -50,8 +50,12 @@ java -jar contribution/releasenotes-builder/target/releasenotes-builder-1.0-all.
      -releaseNumber "$CS_RELEASE_VERSION" \
      -githubAuthToken "$READ_ONLY_TOKEN" \
      -generateXdoc \
-     -xdocTemplate $BUILDER_RESOURCE_DIR/templates/xdoc_freemarker.template \
-     -publishXdoc -publishXdocWithPush
+     -xdocTemplate $BUILDER_RESOURCE_DIR/templates/xdoc_freemarker.template
+
+cd ../
+sed -i '12r .ci-temp/xdoc.xml' src/xdocs/releasenotes.xml
+git add . && git commit -m "config: update version to $CS_RELEASE_VERSION"
```

Comment:
Commit message: "doc: release notes for {version}"

Reply:
Changes Done